Lib/fontTools/misc/transform.py

```diff
@@ -216,7 +216,7 @@ def transformVectors(self, vectors):
         xx, xy, yx, yy = self[:4]
         return [(xx * dx + yx * dy, xy * dx + yy * dy) for dx, dy in vectors]
 
-    def translate(self, x: float = 0, y: float = 0):
+    def translate(self, x: float = 0, y: float = 0, before=False):
```

Comment:
The same could be achieved by using the transform objects differently. I don’t quite like the added complexity.

Reply:
Thanks for the review.

Reply:
> Do you think I should add `Rotation` and `RotationAroundCenter` free functions?

Currently we have `Offset` (which I can also redefine as `Translation` if desired) and `Scale`. I can add `Skew` as well for completeness.

Reply:
Maybe yes (for Translation and Skew). 

Not sure about RotationAroundCenter, but if that helps you, why not.

Reply:
Thanks. I'm gonna add Rotation that takes optional `tx` and `ty`.

Reply:
Fixed. I ended up adding `pre_translate` since it was used in multiple places in HVF code and is much faster to code instead of doing `Translation(x, y).transform(t)`.